README.md

```
@@ -1,49 +1,89 @@
 
 ## 🧞 Commands
 
 All commands are run from the root of the project, from a terminal:
 
+| Command           | Action                                       |
+| :---------------- | :------------------------------------------- |
+| `npm install`     | Installs dependencies                        |
+| `npm run dev`     | Starts local dev server at `localhost:4321`  |
+| `npm run build`   | Build your production site to `./dist/`      |
+| `npm run preview` | Preview your build locally, before deploying |
+| `Technology Stack |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 2; Actual: 1; Too few cells, row will be missing data

(MD056, table-column-count)